src/test/java/edu/ucla/library/iiif/auth/handlers/SinaiAccessTokenHandlerIT.java

```diff
@@ -149,6 +140,8 @@ public void testGetTokenNonBrowser(final Vertx aVertx, final VertxTestContext aC
         }).onFailure(aContext::failNow);
     }
 
+    // TODO: add tests for expired cookies
```

Comment:
Did you mean to leave this as an unimplemented TODO?

Reply:
Yeah, I figured it made more sense to implement with SERV-372, since that's up next and it will most likely change whatever I would have done